csxtools/utils.py

```diff
@@ -1,9 +1,6 @@
 import numpy as np
 import time as ttime
-try:
-    from databroker import get_images
-except ImportError:
-    from dataportal import get_images
+from databroker import get_images
```

Comment:
Log in to notebook.nsls2.bnl.gov and spin up a CSX environment. Then just type !which conda in a cell and execute it. That will tell you which conda environment is being run.  

To answer my question above, spin up a CSX environment on jupyterhub and type !conda list -e. Then copy that output here.  Thanks!